site/src/components/FeatureStageBadge/FeatureStageBadge.tsx

```diff
@@ -19,12 +19,14 @@ type FeatureStageBadgeProps = Readonly<
 	Omit<HTMLAttributes<HTMLSpanElement>, "children"> & {
 		contentType: keyof typeof featureStageBadgeTypes;
 		size?: "sm" | "md" | "lg";
+		showTooltip?: boolean;
 	}
 >;
 
 export const FeatureStageBadge: FC<FeatureStageBadgeProps> = ({
 	contentType,
 	size = "md",
+	showTooltip = true, // This is a temporary until the deprecated popover is removed
```

Comment:
```suggestion
	showTooltip = true, // This is temporary until the deprecated popover is removed
```

also, how can we track that this actually gets cleaned up later?

Reply:
We could create an issue but im not sure its worth the trouble. This will get resolved automatically either when the beta badge is removed or this component moves to the new popover. The issue is that the current popover does not work well when it appears too close to the right edge of the screen.